app/src/main/java/com/jerboa/ui/components/comment/reply/CommentReplyActivity.kt

```diff
@@ -70,82 +71,68 @@ fun CommentReplyActivity(
             if (loading) {
                 LoadingBar(padding)
             } else {
-                when (val postRes = postViewModel.postRes) {
-                    is ApiState.Success -> {
-                        val moderators = postRes.data.moderators
+                commentReplyViewModel.replyItem?.let { replyItem ->
```

Comment:
I understand this fix, but not the other. What does being a moderator have to do with an inbox?